### DISCUSSION TITLE
Arcade

### DISCUSSION THREAD

**Discussion by mr-breakfast** [MauritsWilke/HyAPI#25] [General]
Not claiming it yet, but that's likely what id do next (seems like a small one)

**Comment by MauritsWilke** [comment]
You may but I warn you, arcade is one of the biggest ones because of the lot of submodes.
I'd say its on the same level as duels with size 😓 

But you may always give it a shot!

**Reply by mr-breakfast** [reply]
im looking at the api and i cant see any submodes. unless ofc im just looking at it wrong. I dont see more data than what Murder Mystery gave.
